### PR TITLE
fix: project cereal cleanup

### PR DESCRIPTION
**Pull request by BeksOmega**
## The basics

- [X] I branched from **project-cereal**
- [X] My pull request is against **project-cereal**
- [X] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Work on project cereal

### Description

This is a running PR for small changes I need to make as my project finishes up and things get merged in.

Each commit will be stand alone, and can be reviewed individually.

### CODE REVIEW

**Review by alschmiedt — APPROVED** [review]

File: core/serialization/blocks.js [hunk 1]
```diff
@@ -52,9 +52,6 @@ exports.ConnectionState = ConnectionState;
  *     y: (number|undefined),
```
Comment (alschmiedt) [inline core/serialization/blocks.js:52]: Not from this PR, but can we remove the TODO and the `/* eslint-disable no-use-before-define */` above?

Comment (BeksOmega) [reply core/serialization/blocks.js:52]: We can, but lint still isn't fixed, so we'd have to reorganize the file :/ Right now higher level functions are defined before lower level procedures, and we'd have to flip it.

Comment (alschmiedt) [reply core/serialization/blocks.js:52]: Ahhh, makes sense. Do you plan on fixing before merging in to master? If not, then we should just make an issue to update it so we have something to reference in the TODO.

Comment (BeksOmega) [reply core/serialization/blocks.js:52]: I can link it to https://github.com/google/blockly/issues/5160

Comment (alschmiedt) [reply core/serialization/blocks.js:52]: Yup, sounds great : )

**Pull request merged**